app/views/partials/repo_owning_teams.scala.html

```diff
@@ -33,7 +33,7 @@ <h4 class="mb-0">Teams</h4>
                 <ul class="list-unstyled mb-0">
                     @for(teamName <- RepoOwningTeamsHelper.teamNamesSortedByOwningTeam(repo)) {
                         <li class="list-item">
-                            <a id="link-to-team-page-for-@{teamName.toLowerCase.replaceAll(" ", "-")}" href="/teams/@teamName">@if(repo.owningTeams.contains(teamName)) {<strong>@teamName <span class="repo-owner">Owner</span></strong>} else {@teamName}</a>
+                            <a id="link-to-team-page-for-@{teamName.toLowerCase.replaceAll(" ", "-")}" href="/teams/@teamName">@if(repo.owningTeams.contains(teamName)) {<strong>@teamName&nbsp; <span class="badge rounded-pill text-bg-secondary">Owner</span></strong>} else {@teamName}</a>
```

Comment:
```suggestion
                            <a id="link-to-team-page-for-@{teamName.toLowerCase.replaceAll(" ", "-")}" href="/teams/@teamName">@if(repo.owningTeams.contains(teamName)) {<strong>@teamName<span class="badge rounded-pill text-bg-secondary ms-1">Owner</span></strong>} else {@teamName}</a>
```

Reply:
![Screenshot from 2023-11-16 15-19-44](https://github.com/hmrc/catalogue-frontend/assets/1241373/005903c3-0f2a-4fc2-a266-345fe54ae713)

Reply:
I added the space because it didn't look aesthetic when the pill is almost touching the team name. 

Left is without space:
![image](https://github.com/hmrc/catalogue-frontend/assets/91616550/ca480837-baaa-43a6-8413-a87a7b00fc42)

Reply:
oh yeah, we need the space, I'm just saying if you want to use the `ms-1` style instead of the blank space character